.github/workflows/linux-cpu-arm64-build.yml

```diff
@@ -74,13 +74,19 @@ jobs:
         run: |
           docker run --rm \
           --volume $GITHUB_WORKSPACE:/onnxruntime_src \
-          -w /onnxruntime_src ort_genai_linux_arm64_gha bash -c "/usr/bin/cmake --preset linux_gcc_cpu_release"
+          -w /onnxruntime_src ort_genai_linux_arm64_gha bash -c " \
+            export CARGO_HOME=/onnxruntime_src/.cargo RUSTUP_HOME=/onnxruntime_src/.rustup && \
```

Comment:
The installation and env setup should be moved to dockerfile like `tools/ci_build/github/linux/docker/inference/aarch64/default/cpu/Dockerfile`